components/table/copy-paste.tsx

```diff
@@ -82,11 +94,13 @@ export function CopyPaste({
         >
           {copied ? (
             <>
-              <CheckMarkSVG /> copié
+              <CheckMarkSVG />
+              <span> copié</span>
             </>
           ) : (
             <>
-              <CopySVG /> copier
+              <CopySVG />
+              <span> copier</span>
```

Comment:
This is the important bit. We need to enclose conditionally rendered `TextNode` inside an element for the translation to work.